jerry-core/ecma/operations/ecma-exceptions.h

```diff
@@ -28,6 +28,12 @@
  * @{
  */
 
+#ifndef JERRY_NDEBUG
```

Comment:
Is it really the release build of the full profile where we want to disable error messages? I'm wondering whether the idea would be more fit for the compact profile, both debug and release builds. (The full profile should provide full ES5.1 functionality, which also means error messages. IMO)